src/services/logger.js

```diff
@@ -1,6 +1,48 @@
-const winston = require('winston');
+const chalk = require('chalk');
 
-winston.remove(winston.transports.Console);
-winston.add(winston.transports.Console, { colorize: true });
+class Logger {
+  constructor(silent) {
+    this.silent = silent;
+    this.spinner = null;
+  }
 
-module.exports = winston;
+  pauseSpinner() {
```

Comment:
[Spinner] I have some issues with the spinner that I'd like to discuss live:
- Where does it come from?
- Is this necessarily taking a promise like this?
- Pause/continue naming?

Reply:
As discussed together, let's take care of this loader refactoring once our init command is complete and ready to have the whole picture.